ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -1,15 +1,11 @@
 <script>
+  import { getContext } from "svelte";
   import { location, link, push } from "svelte-spa-router";
-
-  /* TODO(rudolfs): fetch the actual org list */
-  import { orgMocks } from "../../lib/orgMocks.js";
   import * as path from "../../lib/path.js";
-  import { session } from "../../src/session.ts";
-
   import { Avatar, Icon, Title } from "../Primitive";
-  import Remote from "./Remote.svelte";
-
   import AddOrgButton from "./Sidebar/AddOrgButton.svelte";
+
+  const session = getContext("session");
```

Comment:
As this is a component in the design system it should not depend on global state. The idea was that the context is set up for screens consistently have access to dependent data when routed to. This component should depend on the things in the form of props and the screen which render them should get them out of the context. Here `orgs` and `identity` , which would also make it clear what data this component needs to be rendered.

Reply:
Makes sense. Moved the data fetching up to `<SidebarLayout>`. Eventually we'll have to move these "smarter" components out of the design system. But that is out of scope for this issue.